zulip_bots/zulip_bots/bots/tictactoe/tictactoe.conf

```diff
@@ -0,0 +1,3 @@
+[tictactoe]
+email=boo_sender@zulip.com
+full_name=foo
```

Comment:
The tictactoe bot doesn't require a conf file. Rather, `email` and `full_name` are normally read from a bot's zuliprc file. See my next comment on how  to make this work with the terminal script.